src/NRedisStack/Json/JsonCommands.cs

```diff
@@ -60,6 +60,13 @@ public bool Set(RedisKey key, RedisValue path, RedisValue json, When when = When
         return true;
     }
 
+    /// <inheritdoc/>
+    public bool SetFile(RedisKey key, RedisValue path, string filePath, When when = When.Always)
```

Comment:
Why is path a RedisValue? Am I missing something - shouldn't this be a string?

Reply:
RedisValue is actually a string.
 This is the type used by StackExchange.Redis.